src/Blazor.Extensions.Logging/Blazor.Extensions.Logging.csproj

```diff
@@ -15,9 +15,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="3.0.0-preview9.19424.4" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.0.0-preview9.19423.4" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="3.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.0.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3-beta1" />

```

Comment:
Do we need this to be Beta?

Reply:
No, we don't; Apparently in one of DNC preview updates that snuck in; I reverted to 12.0.2 and all is OK. When I get a chance I'll try to remove the dependency and use the new System.Text.Json but on first pass I failed miserably; So in the spirit of "what is not broken don't fix" I am leaving Newtonsoft.Json in place for now.